micromamba/tests/test_env.py

```diff
@@ -70,6 +70,20 @@ def export_env():
     return env_name
 
 
+@pytest.mark.parametrize("json_flag", [None, "--json"])
+def test_env_export_from_history(json_flag, export_env):
+    flags = filter(None, [json_flag])
+    output = helpers.run_env("export", "-n", export_env, "--from-history", *flags)
+
+    # json is already parsed
+    ret = output if json_flag else yaml.safe_load(output)
```

Comment:
@jjerphan I added a test and checked this line fails in the current `main` and this tests passes in my branch